### PR TITLE
New DrMp3AudioImporter plugin using dr_mp3.

### PR DESCRIPTION
**Pull request by williamjcm**
'Tis my first time contributing a plugin, so I *might* have made mistakes here and there. 😅 

Review/feedback welcome!

### CODE REVIEW

File: src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+void DrMp3ImporterTest::mono16() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("DrMp3AudioImporter");
+    CORRADE_VERIFY(importer->openFile(Utility::Directory::join(DRMP3AUDIOIMPORTER_TEST_DIR, "mono16.mp3")));
+
+    CORRADE_COMPARE(importer->format(), BufferFormat::Mono16);
+    CORRADE_COMPARE(importer->frequency(), 44100);
+    CORRADE_COMPARE_AS(importer->data().prefix(4),
+        (Containers::Array<char>{Containers::InPlaceInit, {0, 0, 0, 0}}),
```
Comment (mosra) [inline src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp]: I would prefer to test a larger prefix that has some non-zero values as well, this is giving a false sense of security ;)

Comment (williamjcm) [reply src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp]: Agreed. Will work on it today.

Comment (williamjcm) [reply src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp]: Well, I just did some testing with different files, and the first non-zero value occurs at index 2835 for the latest file I generated with Audacity. :/

Comment (williamjcm) [reply src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp]: Found more info. That delay is apparently caused by LAME: http://lame.sourceforge.net/tech-FAQ.txt

Comment (williamjcm) [reply src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp]: Wait, I could just get a slice starting at index `x` and ending at index `y`. 🤦‍♂

Comment (williamjcm) [reply src/MagnumPlugins/DrMp3AudioImporter/Test/DrMp3ImporterTest.cpp]: Done in 4015d1d.

**Comment by mosra** [comment]
Merged as fe8b70943b8919bc3609b55649825f1c2140c769. This was great, thank you a lot!

I have to admit I didn't do a hearing test -- I assume you did ;)

**Comment by williamjcm** [comment]
> I have to admit I didn't do a hearing test -- I assume you did ;)

Yeah, I did test with a bunch of audio files on my end. A game's OST, and a few loose files.

**Pull request closed without merging**